.github/workflows/staging-analysis.yml

```diff
@@ -8,6 +8,9 @@ on:
     paths:
       - 'analysis/**'
```

Comment:
Maybe the trigger for staging should be modified, so that it doesn't run on the release commit, because in that case would create new staging versions, and an additional gh-pages update. I'm not sure if that is a problem, but something to think about.